packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts

```diff
@@ -0,0 +1,98 @@
+import { css } from "@microsoft/fast-element";
+import { display, focusVisible } from "@microsoft/fast-foundation";
+import {
+    accentForegroundActiveBehavior,
+    accentForegroundHoverBehavior,
+    accentForegroundRestBehavior,
+    neutralForegroundRestBehavior,
+} from "../styles/index";
+
+export const BreadcrumbItemStyles = css`
+    ${display("inline-block")} :host {
+        background: transparent;
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        font-size: var(--type-ramp-base-font-size);
+        fill: currentColor;
+        line-height: var(--type-ramp-base-line-height);
+        outline: none;
+    }
+
+    .listitem {
+        align-items: center;
+        display: flex;
+    }
+
+    .separator {
+        margin: 0 12px;
+    }
+
+    .control {
+        background: transparent;
+        box-sizing: border-box;
+        color: ${accentForegroundRestBehavior.var};
+        cursor: pointer;
+        fill: inherit;
+        outline: none;
+        text-decoration: none;
+        white-space: nowrap;
+    }
+
+    .control:hover {
+        color: ${accentForegroundHoverBehavior.var};
+    }
+
+    .control:active {
+        color: ${accentForegroundActiveBehavior.var};
+    }
+
+    .control .content {
+        position: relative;
+    }
+
+    .control .content::before {
+        content: "";
+        display: block;
+        height: calc(var(--outline-width) * 1px);
+        position: absolute;
+        top: calc(1em + 4px);
+        width: 100%;
+    }
```

Comment:
What is this doing and what's the purpose of the absolute positioning with the before pseudo selector on the span inside the anchor? Does that span give us something?

Reply:
This is creating the underline when hovering over the anchor.
![image](https://user-images.githubusercontent.com/37851220/89444258-4a077600-d706-11ea-8950-242ef8228354.png)